QUEST_LV_0100.tsv

```diff
@@ -287,17 +287,17 @@ QUEST_LV_0100_20150317_000286	$Shouldn't a Revelator be heading to Klaipeda inst
 QUEST_LV_0100_20150317_000287	$I feel so relieved to be stationed in the Western Woods.{nl}There are some places where you'd rather die than be stationed.
 QUEST_LV_0100_20150317_000288	$When you see a Goddess Statue, try offering your worship.{nl}Surely something nice will happen.
 QUEST_LV_0100_20150317_000289	$This is a dangerous area. What are you doing here?
-QUEST_LV_0100_20150317_000290	$Thank you.{nl}I better go report to Titas about the roaming Large Kepa.
+QUEST_LV_0100_20150317_000290	$Thank you.{nl}I should report the roaming Large Kepa to Titas.
 QUEST_LV_0100_20150317_000291	$Weird... That is not a type of monster you'd face in a place like this.{nl}It must be hard to travel around.
 QUEST_LV_0100_20150317_000292	$Assemble already?{nl}That's tough. There's still a pile of missions.
 QUEST_LV_0100_20150317_000293	{memo X}$First, defeat the Hanamings around here.{nl}Then, collect their petals.
-QUEST_LV_0100_20150317_000294	$Even if the Goddess arrived, I will not go until the mission is completed.
+QUEST_LV_0100_20150317_000294	$Even if it wasn't about the Revelators, but the Godesses returning instead,{nl}I wouldn't abandon my mission before it's over.
 QUEST_LV_0100_20150317_000295	$Yes. This should be enough.{nl}Here is a pill that will restore your stamina instantly. I have enough of it so I will give you this as a gift.{nl}
 QUEST_LV_0100_20150317_000296	$Stamina is consumed when you move.{nl}You can recover stamina by destroying a Tree Root Crystal or by resting.{nl}But you might not have time for that when you are being chased by monsters.{nl}That's when this pill comes in handy.
 QUEST_LV_0100_20150317_000297	$I heard news from my men and was becoming worried.{nl}All is well if it ended up good.
 QUEST_LV_0100_20150317_000298	$Klaipeda Resident
 QUEST_LV_0100_20150317_000299	$)} {nl}When do you think the Goddess is coming back?{nl}Even if I ask the Revelators, all they say is that they must go to Klaipeda...
-QUEST_LV_0100_20150317_000300	$Are you a Revelator?{nl}Didn't the Goddess appeared in your dream?{nl}I've heard so from the soldiers.
+QUEST_LV_0100_20150317_000300	$Are you a Revelator?{nl}Didn't a Goddess appear to you in a dream?{nl}I've heard so from the soldiers.
```

Comment:
Are you a Revelator?{nl}Didn't a Goddess appear to you in a dream?{nl}I heard so from the soldiers.